### PR TITLE
chore(utils/httpbin): make delay return 200 on contex cancel explicit

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

File: utils/httpbin/httpbin.go [hunk 1]
```diff
@@ -69,8 +69,9 @@ func delayHandler(w http.ResponseWriter, r *http.Request) {
 	case <-r.Context().Done():
 		t.Stop()
```
Comment (Choraden) [inline utils/httpbin/httpbin.go:70]: Since we are here, shouldn't we also defer the timer stop?

Comment (Choraden) [reply utils/httpbin/httpbin.go:70]: Don't mention it. It's a timer, not ticker.

**Review by Choraden — APPROVED** [review]

File: utils/httpbin/httpbin.go (hunk 1 above)
Comment (mmatczuk) [reply utils/httpbin/httpbin.go:70]: Sure, I'll add it.

**Comment by mmatczuk** [comment]
Improved commit messages, added deferred timer Stop() call.

**Pull request merged**